samples/StaticHtmlComposites/index.html

```diff
@@ -5,19 +5,20 @@
 </head>
 <body>
   <!-- height need to be set for composite to fit the layout -->
-  <div id="video-call" style="height: 500px"></div>
-  <div id="chat" style="height: 500px"></div>
-  <script src="./dist/chatComposite.js"></script>
-  <script src="./dist/callComposite.js"></script>
-  <script src="./dist/service.js"></script>
+  <div id="video-call" style="height: 50vh"></div>
+  <div id="chat" style="height: 50vh"></div>
+  <script src="./chatComposite.js"></script>
+  <script src="./callComposite.js"></script>
+  <script src="./service.js"></script>
   <script type="module">
     const { user, token } = await service.createUserWithToken();
+    const guid = await service.createGUID();
```

Comment:
maybe we should use const groupId .. so its more clear what its going to be used for